mcs/build/profiles/unityaot.make

```diff
@@ -0,0 +1,40 @@
+#! -*- makefile -*-
+
+BOOTSTRAP_PROFILE = build
+
+BOOTSTRAP_MCS = MONO_PATH="$(topdir)/class/lib/$(BOOTSTRAP_PROFILE)$(PLATFORM_PATH_SEPARATOR)$$MONO_PATH" $(INTERNAL_CSC)
+MCS = $(BOOTSTRAP_MCS)
+
+profile-check:
+	@:
+
+DEFAULT_REFERENCES = mscorlib
+
+PROFILE_MCS_FLAGS = \
+	-d:NET_1_1 \
+	-d:NET_2_0 \
+	-d:NET_2_1 \
+	-d:NET_3_5 \
+	-d:NET_4_0 \
+	-d:NET_4_5 \
+	-d:MONO \
+	-d:UNITY \
+	-d:UNITY_AOT \
+	-d:MOBILE,MOBILE_LEGACY \
+	-d:DISABLE_COM \
+	-d:WIN_PLATFORM \
```

Comment:
Fine for now, but if the aot profile will be cross platform the windows defines likely don't make sense.